app/controllers/people_controller.rb

```diff
@@ -0,0 +1,22 @@
+class PeopleController < ApplicationController
+  def index
+    @displaypeople = Person.all
+    render :index
+    # This isn't necessary since it's implicit. However
+    # we should prefer clarity.
+  end
+
+  def show
+    @id = params[:id]
+    @person = Person.find(@id)
+    render :show
+  end
+
+  def tasks
+    @id = params[:id]
```

Comment:
It looks like `@id` isn't used in the view so this should be a local variable rather than an instance variable.